### PR TITLE
support types and add types case

### PR DESCRIPTION
**Pull request by kidyfirst**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,13 +30,16 @@
...
+    "http": "^0.0.0",
     "pre-commit": "^1.2.2",
     "snazzy": "^8.0.0",
     "standard": "^12.0.1",
-    "tap": "^12.5.3"
+    "tap": "^12.5.3",
+    "typescript": "^3.4.3"
   },
   "dependencies": {
+    "@types/node": "^11.13.2",
```
Comment (mcollina) [inline package.json]: Can you move this to devDependencies?

File: package.json [hunk 2]
```diff
@@ -28,13 +30,16 @@
   },
   "homepage": "https://github.com/fastify/fastify-autoload#readme",
   "devDependencies": {
-    "fastify": "^2.0.0",
+    "fastify": "^2.2.0",
+    "http": "^0.0.0",
```
Comment (mcollina) [inline package.json]: this is not a thing, remove it

**Comment by mcollina** [comment]
@Ethan-Arrowood @delvedor would you mind to take a look?

File: package.json (hunk 1 above)
Comment (kidyfirst) [reply package.json]: ok

File: package.json (hunk 2 above)
Comment (kidyfirst) [reply package.json]: have removed

**Pull request merged**